src/azure-cli/azure/cli/command_modules/acs/_help.py

```diff
@@ -517,7 +517,7 @@
     short-summary: The resource ID of an existing Log Analytics Workspace to use for storing monitoring data.
 examples:
   - name: Enable Kubernetes addons. (autogenerated)
-    text: az aks enable-addons --addons virtual-node --name MyManagedCluster --resource-group MyResourceGroup
+    text: az aks enable-addons --addons virtual-node --name MyManagedCluster --resource-group MyResourceGroup --subnet MySubnetName
```

Comment:
Why do we need to update this `autogenereated` example ?

Reply:
Because it doesn't contain all the required parameters. Please see #12871

Reply:
Add @haroldrandom to review the Aladdin part.